5_databases/week2/part3.sql

```diff
@@ -0,0 +1,34 @@
+-- Active: 1699187035090@@127.0.0.1@3306@HYF_lesson2
+USE hyf_lesson2;
+
+--Get all the tasks assigned to users whose email ends in @spotify.com
+
+SELECT * 
+FROM user 
```

Comment:
Here you should use the user_task many-to-many table to make relations like you do below so you get the task and not the user